frontend/src/app/draft/page.tsx

```diff
@@ -0,0 +1,102 @@
+"use client";
+
+import { axiosInstance } from "@services/api-client";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { Heading } from "@ui/Heading";
+import { LinkButton } from "@ui/LinkButton";
+
+interface dataValues {
+  id: number;
+  title: string;
+  desc: string;
+  author: string;
+}
+
+const Draft = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    async function getBooks() {
+      const response = await axiosInstance.get("/api/books/10/pages");
+      setData(response.data);
+    }
+    getBooks();
+  }, []);
+
```

Comment:
We should make customize hook to get data from API. That way if we can reuse it in another component if needed. I would suggest following the folder structure as per the dev branch.